docs/DeveloperGuide.adoc

```diff
@@ -98,13 +110,13 @@ image::LogicClassDiagram.png[]
 *API* :
 link:{repoURL}/src/main/java/seedu/address/logic/Logic.java[`Logic.java`]
```

Comment:
This looks like the same diagram as before, not sure if it'll be good to replace one of the two.